Userland/Libraries/LibGfx/SkiaBackendContext.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Noncopyable.h>
+
+#ifdef AK_OS_MACOS
+#    include <LibCore/MetalContext.h>
+#endif
+
+#ifdef USE_VULKAN
+#    include <LibCore/VulkanContext.h>
+#endif
```

Comment:
Does any code other than LibGfx here now use these contexts? If so, can we move them to LibGfx?

Does LibWebView do them? If so, does it already depend on LibGfx?

Reply:
I think we can move Vulkan and Metal contexts into LibGfx and it will be ok.

> Does LibWebView do them? If so, does it already depend on LibGfx?

yes, LibWebView depends on LibGfx but it does not need Vulkan or Metal.

Reply:
ok, I moved both Vulkan and Metal context from LibCore to LibGfx.